teflo_datarouter_plugin/helpers.py

```diff
@@ -35,15 +35,31 @@ def validate_compose_payload_content(tar_dir_list):
 
 
 def validate_struc_before_compose(payload_dir):
-    dir_contenct = []
+    """ Validating payload contain results/ dir and is not empty."""
```

Comment:
this will not work if the user is sending only payload for resultsdb, where he just needs the `resultsdb` folder in the payload dir.